CONTRIBUTING.md

```diff
@@ -65,6 +65,22 @@ You can also clean up generated files:
 
 `yarn clean`
 
+### Elm components stories
+Besides creating your Elm story on an Elm file, there is an extra step in order to make it appear on the storybook. 
+You have to refer it on the component's JS story.
+
+```
+-- MyComponent.tsx
+import { loadElmStories } from "@cultureamp/elm-storybook"
+
+// JS stories
+
+loadElmStories("MyComponent (Elm)", module, require("./MyComponent.elm"), [
+  "Default",
+  "Default /w icon",
```

Comment:
To give more visibility here to the engineer's current task of adding a story, can we rename one of these to something more prominent? E.g. "Your New Story Name"?